### PR TITLE
Falcon 2 changlog

### PR DESCRIPTION
**Pull request by tutok**
### Please check if the PR fulfills these requirements ###

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
`CHANGELOG.md` update

**Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)**

**Other information:**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,16 +9,67 @@ Versions marked with a number and date (e.g. Falcon Client v0.1.0 (2018-10-05))
...
+- fixed typos in build log messages ([602](https://github.com/deity-io/falcon/pull/602))
+- added Apollo hooks support (upgrade `apollo-client` to version `2.6.x`) ([#596](https://github.com/deity-io/falcon/pull/596))
+
+### Falcon UI Next
+
+- fixed theme merging ([#551](https://github.com/deity-io/falcon/pull/551))
+
+### Falcon E-commerce UI Kit discontinued
+
+- Breaking change: Falcon E-commerce UI Kit is discontinued, instead we introduce [Falcon Front Kit](#falcon-front-kit-v0.0.1), [Falcon UI Kit](#falcon-ui-kit-v0.0.1), [Falcon Data](#falcon-data-v0.0.1), [Falcon Shop Data](#falcon-shop-data-v0.0.1) and [Falcon Blog Data](#falcon-blog-data-v0.0.1) ([#487](https://github.com/deity-io/falcon/pull/487))
```
Comment (mateuszwozniak) [inline CHANGELOG.md]: The versions here can be changed to 1.0.0 as all of these will get that version with the official release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,16 +9,67 @@ Versions marked with a number and date (e.g. Falcon Client v0.1.0 (2018-10-05))
...
+- added Apollo hooks support (upgrade `apollo-client` to version `2.6.x`) ([#596](https://github.com/deity-io/falcon/pull/596))
+
+### Falcon Blog Data v0.0.1
+
+- initial release
+- added Apollo hooks support (upgrade `apollo-client` to version `2.6.x`) ([#596](https://github.com/deity-io/falcon/pull/596))
 
 ### Falcon Server vNext
 
+- FalconServer/Extensions rework ([#479](https://github.com/deity-io/falcon/pull/479))
```
Comment (mateuszwozniak) [inline CHANGELOG.md]: @Borales can you please add a short info explaning what has been done there?

Comment (Borales) [reply CHANGELOG.md]: addede

File: CHANGELOG.md (hunk 1 above)
Comment (tutok) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,21 +9,90 @@ Versions marked with a number and date (e.g. Falcon Client v0.1.0 (2018-10-05))
 ### Falcon Client vNext
 
 - changed the way of defining "proxyEndpoints" ([#571](https://github.com/deity-io/falcon/pull/571))
+- fixed typos in build log messages ([602](https://github.com/deity-io/falcon/pull/602))
```
Comment (Borales) [inline CHANGELOG.md]:
```suggestion
- fixed typos in build log messages ([#602](https://github.com/deity-io/falcon/pull/602))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,21 +9,90 @@ Versions marked with a number and date (e.g. Falcon Client v0.1.0 (2018-10-05))
...
+- added Apollo hooks support (upgrade `apollo-client` to version `2.6.x`) ([#596](https://github.com/deity-io/falcon/pull/596))
+- improved code redistribution across chunks ([#619](https://github.com/deity-io/falcon/pull/619))
+- fixed self cached service worker (`/sw.js`) ([#626](https://github.com/deity-io/falcon/pull/626))
+- upgrade `react` to version `16.10.2` ([#640](https://github.com/deity-io/falcon/pull/640))
```
Comment (Borales) [inline CHANGELOG.md]:
```suggestion
- upgraded `react` to version `16.10.2` ([#640](https://github.com/deity-io/falcon/pull/640))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -9,21 +9,90 @@ Versions marked with a number and date (e.g. Falcon Client v0.1.0 (2018-10-05))
...
+- improved eslint configuration ([#642](https://github.com/deity-io/falcon/pull/642)), ([#639](https://github.com/deity-io/falcon/pull/639)), ([#633](https://github.com/deity-io/falcon/pull/633))
+- upgrade `css-loader` to version `3.2.0` ([#620](https://github.com/deity-io/falcon/pull/620))
```
Comment (Borales) [inline CHANGELOG.md]:
```suggestion
- upgraded `css-loader` to version `3.2.0` ([#620](https://github.com/deity-io/falcon/pull/620))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -9,21 +9,90 @@ Versions marked with a number and date (e.g. Falcon Client v0.1.0 (2018-10-05))
...
+- fixed `/graphQL` proxy error handling ([#632](https://github.com/deity-io/falcon/pull/632))
+- fixed custom error `500.html` view resolution ([#662](https://github.com/deity-io/falcon/pull/662))
+
+### Falcon UI Next
```
Comment (nimobeeren) [inline CHANGELOG.md]:
```suggestion
### Falcon UI vNext
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -9,21 +9,90 @@ Versions marked with a number and date (e.g. Falcon Client v0.1.0 (2018-10-05))
...
 ### Falcon Server vNext
 
+- Falcon-Server has been fully migrated to [TypeScript](https://github.com/microsoft/TypeScript/) 🎉 ([#479](https://github.com/deity-io/falcon/pull/479))
+  - Changed the flow of Extension Containers, whereby `Extension` base class has been removed in favor of a cleaner code ([read more](https://falcon.deity.io/docs/falcon-server/extensions))
+  - Now every `schema.graphql` file needs to be placed in the root of the package
+  - Custom `Extenstion` can deliver its own `schemaDirectives` now which will be properly merged into the main Schema
+  - Improved BackendConfig fetching technique (asynchronously in parallel)
 - improved Cache calls by tracking simultaneous requests with the same cache-key ([#557](https://github.com/deity-io/falcon/pull/557))
 - introduced `@cacheId` directive to assist to `@cache` for generating cache tags ([#608](https://github.com/deity-io/falcon/pull/608))
+- falsy value check when extracting a value by `fieldName` from the provided `sourceValue` ([#607](https://github.com/deity-io/falcon/pull/607))
```
Comment (Borales) [inline CHANGELOG.md]:
```suggestion
- fixed falsy value check when extracting a value by `fieldName` from the provided `sourceValue` ([#607](https://github.com/deity-io/falcon/pull/607))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -9,21 +9,90 @@ Versions marked with a number and date (e.g. Falcon Client v0.1.0 (2018-10-05))
...
+
+### Falcon UI Next
+
+- fixed theme merging ([#551](https://github.com/deity-io/falcon/pull/551))
+- fixed duplicated conditions when extracting themed css ([#611](https://github.com/deity-io/falcon/pull/611))
+- fixed incompatible `css` prop typings ([#645](https://github.com/deity-io/falcon/pull/645))
+
+### Falcon E-commerce UI Kit discontinued
+
+- Breaking change: Falcon E-commerce UI Kit is discontinued, instead we introduce [Falcon Front Kit](#falcon-front-kit-v1.0.0), [Falcon UI Kit](#falcon-ui-kit-v1.0.0), [Falcon Data](#falcon-data-v1.0.0), [Falcon Shop Data](#falcon-shop-data-v1.0.0) and [Falcon Blog Data](#falcon-blog-data-v1.0.0), (see rework [log](https://github.com/deity-io/falcon/blob/a5274623c148b9b0c8cc1d17254d7c89395955da/packages/falcon-ecommerce-uikit/uikit.md)) ([#487](https://github.com/deity-io/falcon/pull/487))
```
Comment (nimobeeren) [inline CHANGELOG.md]:
```suggestion
- Breaking change: Falcon E-commerce UI Kit has been discontinued, instead we introduce [Falcon Front Kit](#falcon-front-kit-v1.0.0), [Falcon UI Kit](#falcon-ui-kit-v1.0.0), [Falcon Data](#falcon-data-v1.0.0), [Falcon Shop Data](#falcon-shop-data-v1.0.0) and [Falcon Blog Data](#falcon-blog-data-v1.0.0), (see rework [log](https://github.com/deity-io/falcon/blob/a5274623c148b9b0c8cc1d17254d7c89395955da/packages/falcon-ecommerce-uikit/uikit.md)) ([#487](https://github.com/deity-io/falcon/pull/487))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -9,21 +9,90 @@ Versions marked with a number and date (e.g. Falcon Client v0.1.0 (2018-10-05))
...
+- added convention for validation error messages translation ([#585](https://github.com/deity-io/falcon/pull/585))
+- fixed synchronisation of account and sign in icon with customer logging in and out state ([#521](https://github.com/deity-io/falcon/pull/521))
+- added Apollo hooks support (upgrade `apollo-client` to version `2.6.x`) ([#596](https://github.com/deity-io/falcon/pull/596))
+- improved dynamic url resolution ([#618](https://github.com/deity-io/falcon/pull/618))
+
+### Falcon UI Kit v1.0.0
+
+- initial release
+- added Apollo hooks support (upgrade `apollo-client` to version `2.6.x`) ([#596](https://github.com/deity-io/falcon/pull/596))
+- fixed displaying multiple street numbers in `AddressDetails` component ([#614](https://github.com/deity-io/falcon/pull/614))
```
Comment (nimobeeren) [inline CHANGELOG.md]:
```suggestion
- fixed warning when `AddressDetails` receives non-unique values ([#614](https://github.com/deity-io/falcon/pull/614))
```

**Pull request merged**